### ISSUE TITLE
When are the images converted to range(0,1) in pre-trained resnet50_v2 on imagenet?

### ISSUE THREAD

**Issue by jonbakerfish** [apache/incubator-mxnet#10935]
From the Gluon Model Zoo:

> All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (N x 3 x H x W), where N is the batch size, and H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]

However, when I look at the train scripts for resnet50_v2 (i.e. https://github.com/apache/incubator-mxnet/blob/master/example/gluon/image_classification.py and https://github.com/apache/incubator-mxnet/blob/master/example/gluon/data.py)
 There is no code line showing that the image is converted to range (0,1) (i.e. `image = image/255`) in the `get_imagenet_iterator`. 
It seems that the only explanation is that the images are loaded in [0, 1] range by the [ImageFolderDataset](https://github.com/apache/incubator-mxnet/blob/b011eccd17d660b2f1ecf901e0db0007706df35c/python/mxnet/gluon/data/vision/datasets.py#L264). 
Please correct me if I'm wrong.

**Comment by chinakook** [comment]
https://github.com/apache/incubator-mxnet/blob/b011eccd17d660b2f1ecf901e0db0007706df35c/example/gluon/data.py#L63

**Comment by jonbakerfish** [comment]
Hi @chinakook,
If you run this code:

    im = mx.nd.random_uniform(shape=(3, 10, 10)) * 255
    im_ = mx.nd.image.normalize(im, mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225))
    print im_.min(),im_.max() 

you will get something like:

    [-1.89502692]
    <NDArray 1 @cpu(0)> 
    [ 1131.16967773]
    <NDArray 1 @cpu(0)>

I think the mean and std are for images which are in range (0,1), not for the images in range (0,255). So before doing `image.normalize`, `im=im/255` should be done somewhere else.

**Comment by chinakook** [comment]
```image = mx.nd.image.to_tensor(image)``` will do that.

**Comment by jonbakerfish** [comment]
Hi @chinakook ,
Thanks for pointing out. Also, I found that the [mxnet.gluon.data.vision.transforms.ToTensor](https://mxnet.incubator.apache.org/_modules/mxnet/gluon/data/vision/transforms.html#ToTensor) use `F.image.to_tensor` internally.
